ModuleSettings.cs

```diff
@@ -9,6 +9,12 @@ public enum MarkerClipboardConsentLevel {
         OnlyWhenInteractedWith,
         Never
     }
+    
+    public enum VisibilityLevel {
+        Default,
```

Comment:
We may wish to revise this wording at some point to better represent it as default being whatever the marker or trail itself has chosen to use, but I can't think of a succinct name to call it.